crates/control/src/search_suggestor.rs

```diff
@@ -151,3 +161,45 @@ impl IndexMut<Point2<Field>> for Heatmap {
         &mut self.map[heatmap_point]
     }
 }
+
+fn get_rule_hypotheses(
+    primary_state: PrimaryState,
+    filtered_game_controller_state: FilteredGameControllerState,
+    field_dimensions: FieldDimensions,
+) -> Vec<Point2<Field>> {
+    let kicking_team_half =
+        conservative_kicking_team_half(filtered_game_controller_state.kicking_team);
+
+    match (primary_state, filtered_game_controller_state.sub_state) {
+        (PrimaryState::Ready, Some(SubState::PenaltyKick)) => {
```

Comment:
Should this also include the primary state Set?

Reply:
I'm not sure about the exact ball search mechanics, but during set no one should be doing ball search anyways.

@JonasHolz do you think it makes sense to apply rule ball positions during set also?

Reply:
macht zu mir sinn